src/Features/CSharp/Portable/AddFileBanner/CSharpAddFileBannerNewDocumentFormattingProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.AddFileBanner;
+using Microsoft.CodeAnalysis.CSharp.CodeGeneration;
+using Microsoft.CodeAnalysis.CSharp.FileHeaders;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.FileHeaders;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.AddFileBanner
+{
+    [Export(typeof(INewDocumentFormattingProvider)), Shared]
```

Comment:
I don't think this is right, because it doesn't specify a language, but I'm assuming `ExportLanguageService` won't work with the `ImportMany` that I need to use to get all of the providers, since there is no way to get multiple language services from a document.